### ISSUE TITLE
PyBrain RuntimeWarning: overflow encountered when using Gaussian layer - works for Sigmoid

### ISSUE THREAD

**Issue by SchmidtQQ** [pybrain/pybrain#204]
I have also posted this problem @ StackOverflow: http://stackoverflow.com/questions/38011905/pybrain-runtimewarning-overflow-encountered-when-using-gaussian-layer-works-f 

I am using pybrain (http://pybrain.org/) for training my feed forward neural network but for some reason I am getting:
`c:\Python27\lib\site-packages\pybrain-0.3.3-py2.7.egg\pybrain\supervised\trainers\backprop.py:113: 
RuntimeWarning: overflow encountered in square
error += 0.5 * sum(outerr ** 2) 
c:\Python27\lib\site-packages\numpy\core\numeric.py:1093: RuntimeWarning: overflow encountered in multiply
return multiply(a.ravel()[:, newaxis], b.ravel()[newaxis,:], out)
c:\Python27\lib\site-packages\pybrain-0.3.3-py2.7.egg\pybrain\structure\parametercontainer.py:81: RuntimeWarning: invalid value encountered in multiply
self._derivs *= 0
...
File "c:\Python27\lib\site-packages\pybrain-0.3.3-py2.7.egg\pybrain\supervised\trainers\backprop.py", line 248, in trainUntilConvergence
raise Exception("Training produced NaN results")
Exception: Training produced NaN results`

I have seen an earlier posts regarding this, but I ensured that all the components in the training data set are normalized and all have the same dimension.

Funny enough, the code works when I change from Gaussian layer to Sigmoid layer. Any idea why might this be happening and how to fix this?

I am using 64 bit python, and still explicity converting my input data set to np.float64.

Here is a snippet from my code:

`...
trainingDataSet, validationDataSet = myDataSet.splitWithProportion(0.75)
mytrainer = BackpropTrainer(myNeuralNetwork, myDataSet, 
                            batchlearning=False, learningrate = 0.001, 
                            momentum = 0.99, weightdecay=0.001)
mytrainer.trainUntilConvergence(verbose=True,
                          trainingData=trainingDataSet,
                          validationData=validationDataSet,
                          maxEpochs=100,
                          continueEpochs=10)
...`

**Comment by qq363866078** [comment]
try to change the parameter of you gaussian function, make sure it has the same value range with sigmoid,
or you can change you objective values.
